griptape/drivers/web_scraper/base_web_scraper_driver.py

```diff
@@ -0,0 +1,8 @@
+from abc import ABC, abstractmethod
+from typing import Optional
+
+
+class BaseWebScraperDriver(ABC):
+    @abstractmethod
+    def scrape_url(self, url: str, *args, **kwargs) -> str:
```

Comment:
Should probably return a `TextArtifact`.

Reply:
Updated